app/src/main/java/pm/gnosis/heimdall/helpers/AddressHelper.kt

```diff
@@ -55,4 +61,8 @@ class AddressHelper @Inject constructor(
         imageView?.setAddress(address)
         return listOf(buildAddressInfoSingle(addressView, nameView, address).subscribeBy(onError = Timber::e))
     }
+
+    companion object {
+        private val MULTI_SEND_LIB = BuildConfig.MULTI_SEND_ADDRESS.asEthereumAddress()!!
```

Comment:
is multisend a lib or smart contract?

Reply:
https://github.com/gnosis/safe-contracts/blob/development/contracts/libraries/MultiSend.sol

It is a smart contract that is a library for the Safe (see https://gnosis-safe.readthedocs.io/en/latest/contracts/architecture.html#libraries)